sdk/storage/storage-internal-avro/package.json

```diff
@@ -12,86 +12,103 @@
   "bugs": {
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
-  "main": "./dist/index.js",
-  "module": "dist-esm/src/index.js",
+  "main": "./dist/commonjs/index.js",
+  "module": "./dist/esm/index.js",
   "keywords": [
     "azure",
     "cloud"
   ],
-  "browser": {
-    "./dist-esm/storage-internal-avro/src/index.js": "./dist-esm/storage-internal-avro/src/index.browser.js",
-    "./dist-esm/storage-internal-avro/src/utils/utils.node.js": "./dist-esm/storage-internal-avro/src/utils/utils.browser.js",
-    "./dist-esm/storage-internal-avro/test/utils/index.js": "./dist-esm/storage-internal-avro/test/utils/index.browser.js",
-    "fs": false,
-    "os": false,
-    "process": false
-  },
-  "types": "./types/latest/storage-internal-avro.d.ts",
+  "browser": "./dist/browser/index.js",
+  "types": "./dist/commonjs/index.d.ts",
   "engines": {
     "node": ">=18.0.0"
   },
   "files": [
     "dist/",
-    "dist-esm/src/",
-    "types/latest",
     "README.md",
     "LICENSE"
   ],
   "scripts": {
-    "build": "npm run clean && npm run build:es6 && dev-tool run bundle && dev-tool run extract-api",
+    "build": "npm run clean && dev-tool run build-package && dev-tool run extract-api",
     "build:es6": "tsc -p tsconfig.json",
     "build:samples": "echo Skipped.",
     "build:test": "npm run build:es6 && dev-tool run bundle",
     "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "dev-tool run vendored rimraf --glob dist dist-* types temp statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
     "docs": "echo Skipped.",
-    "extract-api": "tsc -p . && dev-tool run extract-api",
+    "extract-api": "dev-tool run build-package && dev-tool run extract-api",
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "dev-tool run test:node-js-input --no-test-proxy -- -t 300000  \"dist-esm/test/*.spec.js\" \"dist-esm/test/node/*.spec.js\"",
+    "integration-test:browser": "npm run clean && dev-tool run build-package && dev-tool run build-test && dev-tool run test:vitest --browser",
+    "integration-test:node": "dev-tool run test:node-js-input --no-test-proxy -- --exclude 'test/snippets.spec.ts' -t 300000  \"dist-esm/test/*.spec.js\" \"dist-esm/test/node/*.spec.js\"",
```

Comment:
this command should also be replaced with the vitest one.

Reply:
Fixed.